doc/index.rst

```diff
@@ -21,3 +21,8 @@ Welcome to ESP8266 Arduino Core's documentation!
    Stack Dump <Troubleshooting/stack_dump>
    Using with Eclipse <eclipse/eclipse>
    Changelog <changes>
+
+-Other languages: 
+ATTENTION !!! All communications (issues and pull request) regarding the ESP8266 Arduino core should always be done in English, as it is the official language of the project. Requests are not allowed in any other language.
```

Comment:
Please use Sphinx markup instead of writing 'ATTENTION', especially in all caps. See http://www.sphinx-doc.org/en/stable/markup/para.html (`.. attention::`)

Reply:
Also, I don't think this paragraph is particularly useful here. You probably need to say this in the issue template or in the policy document, but not in the main documentation page.